algorithms.tex

```diff
@@ -1,2 +1,17 @@
+% A description of some of the algorithms in SymPy. The list is not
+% exhaustive.
 
+% The sections here are preliminary. We may end up needing to cut some of
+% this.
 
+\subsection{Numerics}
+
+\subsection{Polynomials}
+
+\subsection{The Risch Algorithm}
```

Comment:
I think it would be good to have, but only if someone can write about it. Otherwise, we should just mention that it's there without describing it, along with the other algorithms that we just mention.